package.json

```diff
@@ -98,20 +90,18 @@
     "react-ace": "9.4.0",
     "react-autosize-textarea": "7.1.0",
     "react-dom": "^17.0.1",
-    "resize-observer-polyfill": "1.5.1",
     "sass": "1.49.8",
     "sass-loader": "9.0.3",
     "style-loader": "1.2.1",
     "tailwindcss": "2.0.2",
     "typescript": "4.5.5",
-    "uuid": "8.3.2",
     "webpack": "5.69.1",
     "webpack-cli": "4.9.2"
   },
   "scripts": {
     "check-licenses": "wp-scripts check-licenses --prod --gpl2",
-    "build": "wp-scripts build --webpack-no-externals",
-    "dev": "wp-scripts start --webpack-no-externals",
```

Comment:
`--webpack-no-externals` made updating the packges important.

Now, the block editor will alias these packages to whatever it's using.